PublicationsRouter/api/v3/Send.md

```diff
@@ -105,28 +105,33 @@ If you are sending only the notification JSON, the request must take the form:
 
 ### 2. Validate Metadata + Package request
 
-If you are sending binary content as well as the metadata, the request must take the form:
+If you are sending binary content as well as the metadata, the request should be formed using [RFC 2387](https://www.ietf.org/rfc/rfc2387.txt):
```

Comment:
Better:
`If you are sending binary content as well as the metadata, then a multi-part request must be formed using [RFC 2387](https://www.ietf.org/rfc/rfc2387.txt):`